scripts/ij.bazelproject

```diff
@@ -1,14 +1,14 @@
 # Setup IntelliJ for Bazel development using the IntelliJ Bazel Plugin.
 # See https://github.com/bazelbuild/intellij for installation instructions.
 directories:
-  src/main
-  src/test
+  .
```

Comment:
The syntax allows removing directories. Can we add:
- bazel-*
to the directories list?